squid/README.md

```diff
@@ -2,33 +2,21 @@
 
 ## Overview
 
-This integration lets you monitor your Squid metrics from the Cache Manager directly in Datadog.
+This check monitors [Squid][9] metrics from the Cache Manager through the Datadog Agent.
 
 ## Setup
 
-Find below instructions to install and configure the check when running the Agent on a host. See the [Autodiscovery Integration Templates documentation][1] to learn how to apply those instructions to a containerized environment.
+Follow the instructions below to install and configure this check on the host Agent. For containerized environments, see the [Autodiscovery Integration Templates][1] for guidance on applying these instructions.
```

Comment:
I'm not sure if "host Agent" is the way to phrase this.  maybe "when installing the agent on a host"? because its the same agent regardless of install / setup method.

Reply:
We use `Host Agent` in other places (including tabs) for example:
https://docs.datadoghq.com/agent/autodiscovery/?tab=hostagent#docker-autodiscovery

Should we change this?